### PR TITLE
New modes at#9 mobility constraint zone enums

### PR DESCRIPTION
**Pull request by nick-knowles**
Revision for GBS compatibilty from Meeting 2021.04.14

### CODE REVIEW

**Review by Aurige — APPROVED** [review]

File: examples/functions/newModes/NewModes-CycleSharingExample.xml [hunk 1]
```diff
@@ -655,6 +655,16 @@
 								</VehicleTypeZoneRestriction>
 							</vehicleRestrictions>
 						</MobilityServiceConstraintZone>
+						<MobilityServiceConstraintZone version="any" id="general_uses">
+							<Name>Normal Use of zone  s</Name>
```
Comment (skinkie) [inline examples/functions/newModes/NewModes-CycleSharingExample.xml:659]: Typo in example

File: xsd/netex_part_5/part5_nd/netex_nm_mobilityServiceConstraintZone_support.xsd [hunk 2]
```diff
@@ -10,7 +10,10 @@
 				<Contributor>Nicholas Knowles</Contributor>
 				<Contributor>Christophe Duquesne</Contributor>
 				<Coverage>Europe</Coverage>
-				<Creator>First drafted for NeTEx version 1.0 CEN TC278 WG3 SG9 Editor Nicholas Knowles.  mailto:schemer@netex.org.uk</Creator>
+				<Creator>First drafted for NeTEx version 1.0 CEN TC278 WG3 SG9 Editor Nicholas Knowles.  mailto:schemer@netex.org.uk</Creator>allUsesAllowed
```
Comment (skinkie) [inline xsd/netex_part_5/part5_nd/netex_nm_mobilityServiceConstraintZone_support.xsd:13]: I don't think this suffix is supposed to be there.

**Comment by skinkie** [comment]
I wonder how it is possible that Travis did not see the error @nick-knowles just fixed.

File: xsd/netex_part_5/part5_nd/netex_nm_mobilityServiceConstraintZone_support.xsd (hunk 2 above)
Comment (Aurige) [reply xsd/netex_part_5/part5_nd/netex_nm_mobilityServiceConstraintZone_support.xsd]: I edited the files for the 2 typos ... Stefan, I think we can direct do such type fix when we spot them

Comment (skinkie) [reply xsd/netex_part_5/part5_nd/netex_nm_mobilityServiceConstraintZone_support.xsd]: Only if the pull request is editable. And to be frank, you approved merging, while you are now editing the pull request.

Comment (Aurige) [reply xsd/netex_part_5/part5_nd/netex_nm_mobilityServiceConstraintZone_support.xsd]: Yes sure ...making them editable is a good point. This one seam to be editable, that's how I did the correct the 2 issues you spotted and that I missed... thanks for spotting them

**Comment by Aurige** [comment]
From that point, is it Ok to merge ? Shall we "merge commits" or "squash merge " (any general recommandation here ?)

**Comment by skinkie** [comment]
@Aurige please read the fine manual.

**Pull request merged**